package.json

```diff
@@ -48,7 +48,7 @@
         "@dhis2/d2-i18n": "^1.0.4",
         "@dhis2/d2-ui-core": "5.1.3",
         "@dhis2/d2-ui-file-menu": "5.1.3",
-        "@dhis2/d2-ui-interpretations": "5.1.3",
+        "@dhis2/d2-ui-interpretations": "^5.2.8",
```

Comment:
Please update all d2-ui components, since they all have the same version, unless there's a particular reason not to.  Also we're using the explicit version pattern for `d2-ui` dependencies here, so let's stick with that unless we want to change all of them.

Reply:
done! updated all `d2-ui` using explicit version.

Reply:
Thanks @joakimia but it looks like you're using `^` version ranges (`^5.2.8`) rather than explicit versions (`5.2.8`).  Minor detail but since that's what we were using here before (and we're not necessarily great at sticking to `semver` guarantees yet) let's stick with it.

Reply:
I've updated all d2-ui packages to use explicit version after moving the div wrapper 👍